about.html

```diff
@@ -0,0 +1,169 @@
+        <div class="navbarcontent">
+          <ul>
+            <li>
+              <a href="#"><i class="fab fa-facebook-square"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fab fa-twitter"></i></a>
+            </li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+        </div>
+      </div>
+    </header>
+    <!-- end header -->
+    <!-- start nav -->
+    <nav class="navbar" id="navbar">
+      <div class="container">
+        <div class="navbarcontainer">
+          <div class="navbarlogo">
+            <a href="#">
+              <img src="./images/mainlogo.jpeg" alt="logoimage" class="logoimage" />
+            </a>
+          </div>
+          <ul class="itemslist" >
+            <li class="listitem"><a href="index.html" class="colored">About</a></li>
+            <li class="listitem"><a href="#">Program</a></li>
+            <li class="listitem"><a href="#">Join</a></li>
+            <li class="listitem"><a href="#">Sponser</a></li>
+            <li class="listitem"><a href="#">News</a></li>
+            <li class="bordereditem">
+              <a href="#" class="borderelement">National Art Council</a>
+            </li>
+          </ul>
+          <div class="bars">
+            <div class="bar bar__one"></div>
+            <div class="bar bar__two"></div>
+            <div class="bar bar__three"></div>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <!-- end nav -->
+    <!-- start main -->
+    <main class="main" id="main">
+      <!-- start sectionone -->
+      <section class="aboutfirstsection" id="firstsection">
+        <div class="overlaysection">
+          <div class="aboutherosection">
+            <h2 class="aboutitle"> RWANDA NATIONAL ARTIST<br> CONFERENCE 2022</h2>
+            <p class="aboutdescription">
+              Bring together ideal artists and managers with minimal effort 
+              and increase individual artist level. Discover the advantages of being mentored 
+              by proffessional people  in the music industry
+            </p>
+          </div>
+        </div>
+      </section>
+      <!-- end sectionone -->
+      <!-- start sectiontwo -->
+      <section class="secondsection about center pd-y" id="sectiontwo about">
+        <div class="container about">
+          <h2 class="heading">
+            RWANDA's Annual Artist Conference 2022
+          </h2>
+          <span class="line"></span>
+          <p>
+            The logo of RWANDA's Annual Artist Conference 2022 was decided
+            through the logo competition from 8.June to 7.July
+          </p>
+          <div class="sectiontwo__container__resposive about">
+            <img
+              src="./images/mainlogo.jpeg"
+              alt="logo3"
+              class="responsive sectiontwo__container__resposive__respo about"
+            />
```

Comment:
- [Optiona] Dear @stevenmukama, Here because of your container image, I can see your design is overflowing on the 375px screen size. Please fix this for a better user experience.